src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectProtocolValidator.cs

```diff
@@ -469,10 +469,10 @@ public virtual HashAlgorithm GetHashAlgorithm(string algorithm)
         /// <summary>
         /// Validates the 'token' or 'code' see: http://openid.net/specs/openid-connect-core-1_0.html
         /// </summary>
-        /// <param name="expectedValue">the expected value of the hash. normally the c_hash or at_hash claim.</param>
-        /// <param name="hashItem">item to be hashed per oidc spec.</param>
-        /// <param name="algorithm">algorithm to use for compute hash.</param>
-        /// <exception cref="OpenIdConnectProtocolException">if expected value does not equal the hashed value.</exception>
+        /// <param name="expectedValue">The expected value of the hash. normally the c_hash or at_hash claim.</param>
+        /// <param name="hashItem">Item to be hashed per oidc spec.</param>
+        /// <param name="algorithm">Algorithm to use for compute hash.</param>
```

Comment:
change: 'Algorithm to use for compute hash.' to 'Algorithm for computing hash over hashItem'